web/angular-wallet/src/app/setup/account/create-passive/create-passive.component.ts

```diff
@@ -12,7 +12,7 @@ import { BurstUtil } from '@burstjs/crypto';
 })
 export class CreatePassiveAccountComponent implements OnInit {
 
-  burstAddressPattern = BurstUtil.burstAddressPattern;
+  burstAddressPattern = burstAddressPattern;
```

Comment:
????

Reply:
Haha good catch. I think this works because the transpiler, but it's smelly and should be changed.